webroot/adm/oncall/participant-summary.html

```diff
@@ -82,6 +82,27 @@ <h5>Participant Opt-out Webhook</h5>
                         <pre id="webhooksStandardOutput"></pre>
                     </div>
                 </div>
+                <div class="row px-2">
+                    <div class="col section">
+                        <h5>Participant Related Keysets</h5>
+                        <div><b>A keyset is related to the participant if it matches below criteria:</b><br>
+                            a. Keysets where allowed_types include any of the clientTypes of the site<br>
+                            b. If this participant has a client key with ID_READER role, we want to rotate all the keysets where allowed_sites is set to null<br>
+                            c. Keysets where allowed_sites include the leaked site<br>
+                            d. Keysets belonging to the leaked site itself<br></div>
+                        <pre class="errorDiv" id="relatedKeysetsErrorOutput"></pre>
+                        <pre id="relatedKeysetsStandardOutput"></pre>
+
+                        <div class="col">
+                            <a href="#" class="btn btn-primary" id="doRotateKeysets">Rotate Keysets</a>
+                            <div>Note: Once rotated, the participant will need to get a new refresh key to be able to decrypt data.</div>
```

Comment:
Suggested wording: 
"Normally, keys don't become active for 24 hours when rotated, which gives participants 24 hours before they need to call sdk.refresh().
However in this case, rotation will make the new keys active immediately. This means participants will not be able to decrypt newly created UID tokens until they have called sdk.refresh(). Note that sdk.refresh() has a recommeded frequency of once per hour".

Reply:
![image](https://github.com/user-attachments/assets/d8fecec9-24f5-489a-853d-8a3b93032556)